app/assets/javascripts/modules/_admin.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Admin module
+ *
+ */
+var admin = (function() {
+
+  return {
+    
+    handleTableLinks: function() {
```

Comment:
Pretty good HTML/CSS solution for the table links: http://stackoverflow.com/questions/17147821/how-to-make-a-whole-row-in-a-table-clickable-as-a-link/23120665#23120665

Another option would be to not worry about making the entire row clickable, keep the background color change `on:hover`, and put the link on a word or icon at one end of the row. I did something similar in compostapp
<img width="1161" alt="screen shot 2016-03-31 at 6 24 35 pm" src="https://cloud.githubusercontent.com/assets/1417199/14193853/1adcf020-f76e-11e5-8fd5-490e3ac184db.png">